dev/com.ibm.ws.kernel.boot.common/src/io/openliberty/checkpoint/spi/CheckpointPhase.java

```diff
@@ -125,6 +126,35 @@ static synchronized void setPhase(String p) {
             }
         }
         THE_PHASE = phase;
+        if (phase != INACTIVE) {
+            phase.addMultiThreadedHook(new CheckpointHook() {
+                @Override
+                public void prepare() {
+                    debug(() -> "Locking locking checkpoint write lock for prepare");
```

Comment:
"Locking locking"?

Reply:
> "Locking locking"?

Should be "Locking checkpoint write lock for prepare"

Reply:
Fixed.